### PR TITLE
call embedly during scraping

### PR DESCRIPTION
**Pull request by yingjieMiao**
@raymondng  @martinraison 

right now we have some duplicated endpoints in embedlyClient. We can consolidate them later (soon).

### CODE REVIEW

File: kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  embedlyStore: EmbedlyStore,
+  clock: Clock
+) extends EmbedlyCommander {
+
+  override def getEmbedlyInfoFromStore(id: Id[NormalizedURI]): Option[ExtendedEmbedlyInfo] = {
+    embedlyStore.get(id) map {_.info}
+  }
+
+  private def needToCallEmbedly(info: StoredExtendedEmbedlyInfo): Boolean = {
+    info.calledEmbedlyAt.plusMonths(6).getMillis < clock.now().getMillis
```
Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala:30]: That's ... quite conservative.

**Comment by raymondng** [comment]
So is the plan to callback to shoebox update image_info and page_info?

**Comment by raymondng** [comment]
Overall lgtm

File: kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala:30]: yeah. @eishay suggested call embedly only once. I am giving it a little buffer. But it's easy to change.

**Comment by yingjieMiao** [comment]
this is for keywords/entities. Kinda separated from the image issue. 
Current plan is to use db callbacks.

File: kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala:30]: Really -- that's interesting. Is the intent to save $$?

**Comment by raymondng** [comment]
You probably know this already, but db callbacks to shoebox can be
problematic. This is outside of the scope of this pull request, but we
should discuss this.

File: kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/embedly/EmbedlyCommander.scala:30]: there are two parts: on demand part should always be executed. rescraping part, I don't know. We need to look up the pricing table.

**Pull request merged**